src/main/java/com/karl/http/HTTPStatus.java

```diff
@@ -0,0 +1,20 @@
+package com.karl.http;
+
+public enum HTTPStatus {
+  Ok("200", "OK"),
+  NotFound("404", "Not Found"),
+  MethodNotAllowed("405", "Not Found");
+
+  public String code;
+  public String reasonPhrase;
+
+  private HTTPStatus(String code, String reasonPhrase) {
+    this.code = code;
+    this.reasonPhrase = reasonPhrase;
+  }
+
+  @Override
+  public String toString() {
```

Comment:
I like that you are overriding the toString function because that is naturally what a developer would want to use to stringify it.